index.html

```diff
@@ -14,17 +15,17 @@
             alt="Логотип cайта туристического городка в штате Аризона.">
         </a>
         <ul class="navigation-list">
-          <li class="navigation-item">
+          <li class="navigation-item bold-style">
```

Comment:
давай переименуем  text-bold
```suggestion
          <li class="navigation-item text-bold">
```

Reply:
исправил